apps/met-anomalycor/server/main.js

```diff
@@ -449,18 +449,12 @@ const doCurveParams = function () {
         );
     }
 
-    var defaultGroup;
-    var defaultDB;
-    if (Object.keys(dbGroupMap).indexOf("NOAA NCEP") !== -1 && dbGroupMap["NOAA NCEP"].indexOf("mv_gfs_grid2grid_metplus") !== -1) {
-        // special default parameters for the NWS METexpress
-        defaultGroup = "NOAA NCEP";
-        defaultDB = "mv_gfs_grid2grid_metplus";
-    } else {
-        // default to the GSL g2g database if it exists, otherwise default to whatever is first alphabetically
-        defaultGroup = (Object.keys(dbGroupMap).indexOf("NO GROUP") !== -1) ? "NO GROUP" : Object.keys(dbGroupMap)[0];
-        defaultDB = (dbGroupMap[defaultGroup].indexOf("mv_gsl_global_g2g") !== -1) ? "mv_gsl_global_g2g" : dbGroupMap[defaultGroup][0];
-    }
-    var defaultModel = (Object.keys(modelOptionsMap[defaultDB]).indexOf("GFS") !== -1) ? "GFS" : Object.keys(modelOptionsMap[defaultDB])[0];
+    var requestedGroup = matsCollections.Settings.findOne({}).appDefaultGroup;
+    var defaultGroup = (Object.keys(dbGroupMap).indexOf(requestedGroup) !== -1) ? requestedGroup : Object.keys(dbGroupMap)[0];
+    var requestedDB = matsCollections.Settings.findOne({}).appDefaultDB;
+    var defaultDB = (dbGroupMap[defaultGroup].indexOf(requestedDB) !== -1) ? requestedDB : dbGroupMap[defaultGroup][0];
+    var requestedModel = matsCollections.Settings.findOne({}).appDefaultModel;
+    var defaultModel = (Object.keys(modelOptionsMap[defaultDB]).indexOf(requestedModel) !== -1) ? requestedModel : Object.keys(modelOptionsMap[defaultDB])[0];
```

Comment:
Looks great! 

I found the comment in the removed code block really helpful for understanding what `defaultGroup`/`defaultDB`/`defaultModel` were doing. Would it be possible to add a similar comment back in to the apps' `main.js` files?